public/app/core/components/AppChrome/History/HistoryWrapper.tsx

```diff
@@ -78,127 +82,205 @@ function HistoryEntryAppView({ entry, isSelected, onClick }: ItemProps) {
-            name={isExpanded ? 'angle-down' : 'angle-right'}
-            onClick={() => setIsExpanded(!isExpanded)}
-            aria-label={expandedLabel}
-            className={styles.iconButton}
+    <span className={styles.marginBottom}>
+      <Stack direction="column" gap={1}>
+        <Stack alignItems="baseline">
+          {views.length > 0 ? (
+            <IconButton
+              name={isExpanded ? 'angle-down' : 'angle-right'}
+              onClick={() => setIsExpanded(!isExpanded)}
+              aria-label={expandedLabel}
+              className={styles.iconButton}
+            />
+          ) : (
+            <Space h={2} />
+          )}
+          <Icon
+            size="sm"
+            name={isSelected ? 'circle-mono' : 'circle'}
+            aria-label={entryIconLabel}
+            className={isExpanded ? styles.iconButtonDot : styles.iconButtonCircle}
           />
-        ) : (
-          <Space h={2} />
-        )}
-
-        <Card
-          onClick={() => {
-            store.setObject('CLICKING_HISTORY', true);
-            onClick();
-          }}
-          href={url}
-          isCompact={true}
-          className={isSelected ? undefined : styles.card}
-        >
-          <Stack direction="column">
-            <div>
-              {breadcrumbs.map((breadcrumb, index) => (
-                <Text key={index}>
-                  {breadcrumb.text} {index !== breadcrumbs.length - 1 ? '> ' : ''}
-                </Text>
-              ))}
-            </div>
-            <Text color="secondary">{moment(time).format('h:mm A')}</Text>
-            {sparklineData && (
-              <Sparkline
-                theme={theme}
-                width={240}
-                height={40}
-                config={{
-                  custom: {
-                    fillColor: 'rgba(130, 181, 216, 0.1)',
-                    lineColor: '#82B5D8',
-                  },
-                }}
-                sparkline={{
-                  y: {
-                    type: FieldType.number,
-                    name: 'test',
-                    config: {},
-                    values: sparklineData.values,
-                    state: {
-                      range: {
-                        ...sparklineData.range,
+          <Card
+            onClick={() => {
+              store.setObject('CLICKING_HISTORY', true);
+              onClick();
+            }}
+            href={url}
+            isCompact={true}
+            className={isSelected ? styles.card : cx(styles.card, styles.cardSelected)}
+          >
+            <Stack direction="column">
+              <div>
+                {breadcrumbs.map((breadcrumb, index) => (
+                  <Text key={index}>
+                    {breadcrumb.text} {index !== breadcrumbs.length - 1 ? '> ' : ''}
+                  </Text>
+                ))}
+              </div>
+              <Text variant="bodySmall" color="secondary">
+                {moment(time).format('h:mm A')}
```

Comment:
This should probably take locale into account, right?

Reply:
🤔 Would it work using `moment.locale()`?